students/kuhnbt/lesson7/html_render.py

```diff
@@ -0,0 +1,174 @@
+"""Hi Natasha and Rob, I didn't totally understand the note in the 
+assignment about submitting attributes to the constructor for an Element
+ class. I understand the point about how passing a reserved word like class
+would cause an error, but the assigment description seemed to suggest 
+using **kwargs to collect user-defined attributes, which I don't think
+would solve the problem because a user could still pass a reserved word. 
+Then in the run_html_render script the code passes strings of attributes 
+which doesn't match what was talked about in the assignment description.
+Anyway, I decided to require the user to pass a dict of strings which 
+seemed to me like the simplest solution. Let me know if I totally 
+missed the point!"""
```

Comment:
(Please let me know if I misunderstood your question)
in step 4
```
Extend the Element class to accept a set of attributes as keywords to the constructor, e.g. run_html_render.py

Element("some text content", id="TheList", style="line-height:200%")
```
it is talking about adding html style attributes as keyword arguments -> 
`def __init__(self, content=None, **kwargs)`
kwargs is essentially a dictionary anyway so what you have will work just need to change the signature